### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Keyboard: MAS2.1.1 : Snapshot popup is not accessible using Keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#441]
### Prerequisites

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to ""Start Profiling"" option and invoke it.
5. This will invokes ""Xamarin Profiler"" application."

### Steps to Reproduce

1. Follow the pre-requisites to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. This will invokes Common options window.
3. Navigate to "Start Profiling" button and invoke it. This will launch Allocations profiling page. 
4. Navigate to "Snapshot" Popup using "Snapshot" button present at top "Allocations" table using keyboard.

### Expected Behavior

The user should be able to invoke the "Snapshot" popup.

### Actual Behavior

User is unable to invoke the snapshot popup using keyboard.

### User Impact

User who rely on keyboard will not be able to access the Snapshot popup.

### Screenshots

![8 snapshot](https://user-images.githubusercontent.com/22956130/41719827-d43f46f4-757e-11e8-9669-5d69f31dbe42.png)

### Environment

macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Profiler-MAC
Wipro
